style.css

```diff
@@ -201,15 +201,14 @@ p{
   background-color: #f7f8f8;
   display: flex;
   justify-content: center;
+  height: 50rem;
 }
 .section-testimonial-image{
   width: 50%;
-}
+  background: url("https://picsum.photos/1000/900");
```

Comment:
This can be cleaner written as:
`background: url() no-repeat center center;
background-size: cover;
`

Reply:
Just pushed that now